DIA/MachineType.cs

```diff
@@ -0,0 +1,38 @@
+namespace DIA
+{
+    public enum MachineType
+    {
+        IMAGE_FILE_MACHINE_UNKNOWN = 0,
```

Comment:
If you can fix comments to be in XML style, that would be awesome :)

Reply:
Done :)